casepro/utils/__init__.py

```diff
@@ -185,3 +185,23 @@ def get_language_name(iso_code):
         LANGUAGES_BY_CODE[iso_code] = lang
 
     return LANGUAGES_BY_CODE[iso_code]
+
+
+def humanise_seconds(seconds):
+    days, secs = divmod(seconds, 86500)
```

Comment:
This should be 86400, no? Anyways we can leverage Django's own time formatting classes here, and that's how we already format durations on the replies tab of the partner dashboard, e.g.

```python
def humanize_seconds(seconds):
    now = timezone.now()
    return timeuntil(now + timedelta(seconds=seconds), now)
```

`humanise` should also be `humanize` to match the US spelling used elsewhere